qemu/tests/cfg/balloon_minimum.cfg

```diff
@@ -0,0 +1,23 @@
+- balloon_minimum: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4
+    type = balloon_check
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    minimum_value_check = yes
+    balloon_type = evict
+    test_tags = "evict"
+    kill_vm_on_error = yes
+    variants:
+        - balloon_negative:
```

Comment:
Suggest changing the name pattern to distinguish with the name of top level variant.

Reply:
Modified，thanks